systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -301,6 +302,10 @@ public static boolean isIpv4Family() {
         return IP_FAMILY.contains(IP_FAMILY_DEFAULT);
     }
 
+    public static boolean isIpv6Family() {
+        return IP_FAMILY.contains(IP_FAMILY_DEFAULT);
```

Comment:
Here should be `IP_FAMILY_VERSION_6` :) copy-paste (I will update it when the acceptance testing farm will be done)